CourseGrab/ViewControllers/Settings/SettingsViewController.swift

```diff
@@ -103,8 +118,45 @@ class SettingsViewController: UIViewController {
         signOutButton.contentHorizontalAlignment = .right
         accountStackView.addArrangedSubview(signOutButton)
     }
+    
+    private func leaveFeedback(_ button: UIButton) {
+        let emailAddress = "team@cornellappdev.com"
+        if MFMailComposeViewController.canSendMail() {
+            let mailComposerVC = MFMailComposeViewController()
+            mailComposerVC.mailComposeDelegate = self
+
+            let subject = "CourseGrab Feedback"
```

Comment:
We can probably just say `mailComposerVC.setSubject("CourseGrab Feedback")`. There isn't really a need to introduce a new, short variable that we only use once.